pkg/machine/e2e/basic_test.go

```diff
@@ -53,6 +53,21 @@ var _ = Describe("run basic podman commands", func() {
 		rmCon, err := mb.setCmd(bm.withPodmanCommand([]string{"rm", "-a"})).run()
 		Expect(err).ToNot(HaveOccurred())
 		Expect(rmCon).To(Exit(0))
+
+		// make sure gvproxy is running
+		before, beforeErr := pgrep(gvproxyBinaryName)
+		Expect(beforeErr).ToNot(HaveOccurred())
+		Expect(before).To(ContainSubstring(gvproxyBinaryName))
```

Comment:
prgrep on linux returns a pid not a name so this check fails
with WSL it fails as well because we do not have gvproxy running by default with WSL